packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -18,20 +20,25 @@ class CupertinoActivityIndicator extends StatefulWidget {
   const CupertinoActivityIndicator({
     Key key,
     this.animating: true,
+    this.radius: _kDefaultIndicatorRadius,
   }) : assert(animating != null),
+       assert(radius != null && radius > 0),
```

Comment:
minor thing: generally speaking, it's better to have two separate asserts than an assert with two expressions &&'ed together, because then when the assert fails, you can more immediately tell what exactly the problem is.
(Applies to various asserts in this PR.)

Reply:
done

Reply:
Hmm.  Looks like you missed this one..